.circleci/config.yml

```diff
@@ -10,17 +10,12 @@ jobs:
           name: Code lint
           command: make lint
 
-      - setup_remote_docker
+      - setup_remote_docker:
+          docker_layer_caching: true
 
       - run:
           name: Create version.json
-          command: |
-            printf '{"commit":"%s","version":"%s","source":"https://github.com/%s/%s","build":"%s"}\n' \
-            "$CIRCLE_SHA1" \
-            "$CIRCLE_TAG" \
-            "$CIRCLE_PROJECT_USERNAME" \
-            "$CIRCLE_PROJECT_REPONAME" \
-            "$CIRCLE_BUILD_URL" | tee version.json
+          command: make version.json
```

Comment:
Maybe we should use those `$CIRCLE_*` variables in the Makefile if we do that

Reply:
I couldn't figure out how to sanely use those `$CIRCLE_` variables in the makefile, so I put this back.